dashboard/src/t5gweb/ui.py

```diff
@@ -79,7 +79,8 @@ def prepare_flask_request(request):
         "post_data": request.form.copy(),
     }
 
-
+def get_timestamp():
```

Comment:
Personally, I think this behavior can be reverted. IMO running `redis_get("time_stamp")` instead of `get_timestamp()` is more optimal. @dcritch thoughts?

Reply:
yeah, I was confused on what you wanted to do, but after our discussion on the last PR, I get it. no need for unnecessary wrappers if we can avoid them